src/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -21,20 +21,41 @@ export interface CheckboxGroupOption {
     color?: string;
 }
 
-interface CheckboxGroupProps {
+export interface CheckboxGroupProps {
+    /**
+     * The options that should be displayed in the checkbox group.
+     */
     options: CheckboxGroupOption[];
-    allColor?: string;
```

Comment:
I think this prop still should be implemented (not removed). It is applied to the all label via the color prop that is passed first to the `Checkbox` component which internally uses `StyledCheckbox` (receiving the color prop).